oidc-client-app-samples/android-client-app-sample/app/src/main/res/raw/config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Missing license header.

Reply:
Avoided adding a license header since this is a config file in json format.